aten/src/ATen/native/native_functions.yaml

```diff
@@ -5223,11 +5223,20 @@
   device_check: NoCheck   # TensorIterator
   variants: function
 
+- func: fake_quantize_per_tensor_affine.tensor_qparams(Tensor self, Tensor scale, Tensor zero_point, int quant_min, int quant_max) -> Tensor
+  device_check: NoCheck   # TensorIterator
+  variants: function
+
 - func: fake_quantize_per_tensor_affine_cachemask(Tensor self, float scale, int zero_point, int quant_min, int quant_max) -> (Tensor output, Tensor mask)
   variants: function
   dispatch:
     CPU, CUDA: fake_quantize_per_tensor_affine_cachemask
 
+- func: _fake_quantize_per_tensor_affine_cachemask_tensor_qparams(Tensor self, Tensor scale, Tensor zero_point, int quant_min, int quant_max) -> (Tensor output, Tensor mask)
```

Comment:
why don't we have fake_quantize_per_tensor_affine_cachemask.tesnor_qparams?

Reply:
I kept getting undefined function error. I think it doesn't play well with the automatic selection of the backward function.
Hopefully prefixing it with `_` removes it from the user facing API.